CodingChallenges/CC_116_Lissajous/P5/curve.js

```diff
@@ -0,0 +1,37 @@
+class Curve {
+    constructor() {
+        this.path = [];
+        this.current = createVector();
+    }
+
+    setX(x) {
+        this.current.x = x;
+    }
+
+    setY(y) {
+        this.current.y = y;
+    }
+
+    addPoint() {
+        this.path.push(this.current);
+    }
+
+    reset() {
+        this.path = [];
+    }
+
+    show() {
+        stroke(255);
+        strokeWeight(1);
+        noFill();
+        beginShape();
+        this.path.forEach(v => {
```

Comment:
Would you mind using a `for of` loop? I think this is more typical of my examples? I can be convinced otherwise though!

Reply:
Ah, old habits, done.